docs/faq.md

```diff
@@ -13,20 +13,20 @@ new architectures very easily as well!
 Also, PEDA development has been quite idle for a few years now, and many new interesting features a
 debugger can provide simply do not exist.
 
-## What if my GDB is < 8.0 ?
+## What if my GDB is < 10.0 ?
 
 GDB was introduced with its Python support early 2011 with the release of GDB 7. A (very) long way
 has gone since and the Python API has been massively improved, and GEF is taking advantage of them
 to provide the coolest features with as little performance impact as possible.
 
-Currently, GEF is optimized for running against GDB version 8.0+, and Python 3.6+. This allows for a
-best performance and best use of the GDB Python API. However, GEF can run on older versions too,
+Currently, GEF is optimized for running against GDB version 10.0+, and Python 3.10+. This allows for
+a best performance and best use of the GDB Python API. However, GEF can run on older versions too,
```

Comment:
'a best performance' doesn't make sense anyway, we can just say "This allows for the best performance and use of the ..." or similar.